static/js/notifications.js

```diff
@@ -300,6 +335,17 @@ function process_notification(notification) {
         content = message.sender_full_name + content.slice(3);
     }
 
+    // The desktop_inline_notification_data is only used in the client
+    // is desktop app. We send this data down so the desktop app can get
+    // the inline reply from notification and pass it back along with
+    // the reply to send_inline_notification_message event above.
+    var desktop_inline_notification_data = {
+        tag: message.id,
+        type: message.type,
+        to: message.type === 'private' ? message.reply_to : message.stream,
+        topic: topic,
+    };
```

Comment:
Oh, one other questions about this: Do we need all of these fields?  It looks like only `tag` / `message.id` is actually used for passing back to the webapp; is the rest just for displaying the "Reply to ..." widget?

Reply:
(We should probably have a comment above the code on both sides of this interface that makes the API really clear about how the desktop app is allowed to use these data)

Reply:
All of the fields are for webapp code here. We make desktop app pass it back to use with the reply from user because that way we avoid storing and managing the message.
Oh, now I realize I wasn't thinking, the message is already stored in `message_store` so we can get it from there when required. So I am going to drop this hunk of change that passed this data down and create these in the `send_inline_notification_reply` event code.

For the variables names, the reply part sounds good. So now that we only need to rename the event does `send_inline_reply_message` sound good?